tests/testdata/default_config.yaml

```diff
@@ -47,9 +47,9 @@ generate:
       max_ctx_size: 4096
     model_path: /cache/instructlab/models/merlinite-7b-lab-Q4_K_M.gguf
     vllm:
+      gpus: null
```

Comment:
Is there a reason for this move?

Reply:
@booxter guessed that the param might have been manually added in the past. My dev environment linter just made these alphabetical

Reply:
Yep; the `sort_keys` is default when dumping this file through test suite / `make` target.